src/index.html

```diff
@@ -7,12 +8,60 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        src="../src/images/imac.jpeg"
+        alt="imac"
+        class="card__img"
+      />
+      <h1 class="card__name-item">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </h1>
+
+      <div class="card__product-code">Product code: 195434</div>
+      <div class="card__rating">
+        <div class="stars">
```

Comment:
Add the modifier class
```suggestion
        <div class="stars stars--4">
```

Reply:
i also don't understand why i need stars--4?